README.md

```diff
@@ -40,16 +31,15 @@ The resulting bundle:
   <script src="stylelint-bundle.min.js"></script>
   ```
 
-* Use CommonJS `require` or AMD `define` or a global `stylelint` (if you don't use modules) to load the bundled script, then access the `lint` function:
+* Access the `stylelint` global:
 
   ```js
-  // const stylelint = require("stylelint"); // uncomment if you use modules
   stylelint.lint({
     code: "body { color: #000; }",
     config: {
-      syntax: 'sugarss',
+      customSyntax: 'sugarss',
```

Comment:
I'm not really sure if this was a documentation error or we did use a wrong property to define custom syntax.

Reply:
It's an intentionally different name I've used to set the syntax, IIRC. I should have probably used `__openstyles__syntax` or something like that

Reply:
https://github.com/openstyles/stylus/blob/030621462cde7fcc773eb3f627e64fb57a564422/edit/editor-worker.js#L73-L75